qgcresources.qrc

```diff
@@ -3,11 +3,8 @@
         <file alias="opensans">resources/fonts/OpenSans-Regular.ttf</file>
         <file alias="opensans-demibold">resources/fonts/OpenSans-Semibold.ttf</file>
     </qresource>
-
```

Comment:
It guess it's gonna get trashed at some point. The other file is alreaedy trounced anyway.